<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1627</width>
    <height>753</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QListWidget" name="listWidget_2">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QListWidget" name="cellList">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>1000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>1091</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>10</y>
          <width>481</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QListWidget" name="FileList">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>1000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="insertButton">
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QListWidget" name="listWidget_3">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>100</height>
       </size>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1627</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuChart">
    <property name="title">
     <string>Chart</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuChart"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
